### ISSUE TITLE
More Vinaya issues

### ISSUE THREAD

**Issue by brahmali** [suttacentral/bilara-data#3610]
- [x] Kd.16:
Remove paragraph after: pli-tv-kd16:16.2.15.
pli-tv-kd16:15.2.5 to pli-tv-kd16:15.2.27: make part of immediately preceding rule, thus making it bold. 
pli-tv-kd16:16.2.16 to pli-tv-kd16:16.2.39: make part of immediately preceding rule, thus making it bold. 

- [x] Pvr 1.2:
pli-tv-pvr1.2:240.3: change html from <p class='end'>{}</p></article> to <p class='endsutta'>{}</p></article>” (which should make it bold rather than italicised).

- [x] Pvr.1.9:
Indent verse lines: pli-tv-pvr1.9:4.1 + pli-tv-pvr1.9:13.1 (that is, add: <span class='verse-line'>)

- [x] Pvr.2.8:
At the bottom of this one, the navigation button, “next” is Pārājikakaṇḍa rather than something in English, or rather than Katthapaññattivāra.
And then in [pvr2.9](https://suttacentral.net/pli-tv-pvr2.9/en/brahmali), “next” has “Laying-Down-Where?” when it’s about how many offenses…

- [x] Pvr.2.9:
Indent verse lines: pli-tv-pvr2.9:4.1 + pli-tv-pvr2.9:12.1 (that is, add: <span class='verse-line'>)
At https://suttacentral.net/pli-tv-pvr?view=normal&lang=en it shows the card thingy for [pvr2.9](https://suttacentral.net/pli-tv-pvr2.9/en/brahmali) with an untranslated title (and therefore it’s also untranslated in the navigation thingies at the bottom of each page).

- [x] Pvr. 5:
Segment pli-tv-pvr5:84.2 to be formatted like segment pli-tv-pvr5:74.1.

- [x] Pvr. 8:
Move paragraph from after to before segment pli-tv-pvr8:4.2.

- [x] Pvr.10:
pli-tv-pvr10:34.1-36.7 except 35.1-2 formatting: missing verse format.

- [x] Pvr. 11:
move hr from after segment pli-tv-pvr11:52.1 to after segment pli-tv-pvr11:52.2.

- [x] Pvr.15:

Join segment pli-tv-pvr15:13.2 to the following paragraph, same with segments 14.2, 15.2, 17.1, 18.1, 19.1, and 20.1. These last four, together with their paragraphs, should show like ordinary text.

- [x] Pvr.17:
The html is missing an open brace: "pli-tv-pvr17:40.8": "}</p>",
Paragraph should start at segment pli-tv-pvr17:127.13, not at 127.15.
Paragraph should start at segment pli-tv-pvr17:197.4, not at 197.3.
Typo in the card title for [pvr17](https://suttacentral.net/pli-tv-pvr17/en/brahmali).

- [x] Pvr.21:
Move paragraph marker from before to after the following segments: pli-tv-pvr21:15.3, 15.6, 15.9.
Move horizontal rule from before to after the following segments:  pli-tv-pvr21:15.12.

- [x] Add the html "<p class='subrule'>{}</p>" to the following segments:

pli-tv-bi-vb-pc73:1.16
pli-tv-bi-vb-pc75:1.16
pli-tv-bi-vb-pc64:1.16
pli-tv-bi-vb-pc72:1.14
pli-tv-bi-vb-pc63:1.13

- [x] pli-tv-bi-vb-pc7:
Paragraph to begin with segment pli-tv-bi-vb-pc7:1.4 not with segment 1.5 (as it is now).

- [x] pli-tv-bi-vb-pc11: remove segments pli-tv-bi-vb-pc11:2.19 and 2.20 from definition (thereby creating new paragraph without indent as for Bi-Pc 14).
- [x] pli-tv-bi-vb-pc12 & 13: remove segments 2.17 and 2.18 from definitions (thereby creating new paragraph without indent as for Bi-Pc 14).

- [x] pli-tv-pvr7: Start new paragraph after segment pli-tv-pvr7:47.2.
- [x] pli-tv-pvr11: move hr from before to after segment pli-tv-pvr11:52.2.
- [x] pli-tv-pvr13: segment pli-tv-pvr13:6.1 and 6.2 belong to previous paragraph.
- [x] pli-tv-pvr15: segment pli-tv-pvr15:12.2, move paragraph break from before to after this segment.

- [ ] From Tracy:
pli-tv-pvr7:81.24, 81.25 While I think they certainly ought to link to [kd7](https://suttacentral.net/pli-tv-kd7/en/brahmali) 4.1.0 and 5.1.0, suttacentral doesn’t currently go to the right header. It works if the link ends “#pli-tv-[kd7](https://suttacentral.net/pli-tv-kd7/en/brahmali):5.1.0” instead of “#5.1.0” though. I don’t know what happens when this gets printed. (Basically the full segment ID needs to be supplied for the link to fully work if the root is empty for that segment.)
Similarly, pli-tv-pvr7:96.17 and pli-tv-pvr7:96.18 pointing to [kd7](https://suttacentral.net/pli-tv-kd7/en/brahmali) 2.1.0, 3.1.0 and 96.19 to [kd9](https://suttacentral.net/pli-tv-kd9/en/brahmali):5.1.0
Sort of similarly, pli-tv-pvr7:113.31 to #pli-tv-[kd14](https://suttacentral.net/pli-tv-kd14/en/brahmali):12.5.7.7

**Changes to essay at https://suttacentral.net/vinaya-guide-brahmali:**

- [x] “cut-of-point” → cut-off point
In the paragraph right under Content heading: "“The Basket of Monastic Law,” > "the Basket of Monastic Law,”
Americanize:
categorised → categorized
organised
criticising
decentralised
behaviour
specialised
“These section normally” → …sections…
“The Theravāda Khandhakas is a set” is → are
“These include the ordination procedure and the uposatha ceremony, but also” > “These include not only the ordination procedure and the uposatha ceremony, but also”
“Monastic Order” > “Sangha”
p. xiii Final paragraph: "cut-of-point" > "cut-off-point"
p. xiv Final sentence before the sub-heading: "in response the lay people’s" > "in response **to** the lay people's"
p. xv 3rd bullet point: "was completed in first decade of 9th century CE" > "was completed in first decade of **the** 9th century CE"
p.xvii 2nd Paragraph: "such a sexual intercourse or murder" > "such **as** sexual intercourse or murder"
p.xviii 2nd Paragraph: "These section normally take the form" > "These **sections** normally take the form"
p. xix final paragraph: "and we get a glimpse of the Buddha as real person" > "and we get a glimpse of the Buddha as **a** real person"
p.xxi 5th line, last paragraph: "most monastic" > "most monastics".

**Comment by sujato** [comment]
Okay, thanks for these, I will get to them right away, and make sure they are fixed before we do the printed edition.

**Comment by brahmali** [comment]
Sādhu! I've added another issues page, which contain corrections to the introductory essay, the colophon, etc. I've been waiting for bilara 2.0 to do this myself, but it seems it will arrive too late.

**Comment by sujato** [comment]
Yes, it is coming along, but still not usable.

Also, I notice that for the Vinaya "Preface" we still have just placeholder text. I used the Preface to make a short, more personal reflection on the text. It can be just a few paragraphs. I think it's nice to do that, it humanizes the process. 

But if you don't want to write a Preface, we can remove that section.

**Comment by brahmali** [comment]
Yes, I will have to redo the preface, including a proper acknowledgement section. Yeah, so need to fix that. Humanizing is good!

**Comment by sujato** [comment]
Getting to the main items here, let's take them one at a time.

>Kd.16:
Remove paragraph after: pli-tv-kd16:16.2.15.
pli-tv-kd16:15.2.5 to pli-tv-kd16:15.2.27: make part of immediately preceding rule, thus making it bold.
pli-tv-kd16:16.2.16 to pli-tv-kd16:16.2.39: make part of immediately preceding rule, thus making it bold.

Are you sure about this? At the moment they are inconsistent so that needs fixing. But It seems to me the "what five" section is not really part of the rule, more like a vibhanga portion explaining the rule. I'm also wary of nesting structures like this: I'm not sure what will happen if we put a list inside a rule, and I don't know if we do it elsewhere. Also, we shouldn't have too much bold. 

It's a bit of an odd structure, more like an Anguttara sutta in fact. But for me, I'd prefer it if we finished the rule with *Yo vibhajeyya, āpatti thullaccayassa*. What do you think?

**Comment by sujato** [comment]
>Pvr 1.2:
pli-tv-pvr1.2:240.3: change html from

{}

to
{}

” (which should make it bold rather than italicised).

Lol, I've fixed this but if you want to insert code, you have to use backticks:

https://commonmark.org/help/tutorial/09-code.html

**Comment by sujato** [comment]
>Pvr.1.9:
Indent verse lines: pli-tv-pvr1.9:4.1 + pli-tv-pvr1.9:13.1 (that is, add: )

Done this, also in 2.9

**Comment by sujato** [comment]
Regarding the prev/next and other issues for pvr2.8 and 2.9, there was a mistake in the `name` data for these, I have fixed it.

**Comment by sujato** [comment]
Adding checkboxes to the OP so I can keep track of things!

**Comment by sujato** [comment]
>move hr from after segment pli-tv-pvr11:52.1 to after segment pli-tv-pvr11:52.2.

I also fixed similar issues at  pli-tv-pvr16:22.10, pli-tv-pvr16:23.10, pli-tv-pvr21:15.12.

**Comment by sujato** [comment]
>Join segment pli-tv-pvr15:13.2 to the following paragraph, same with segments 14.2, 15.2, 17.1, 18.1, 19.1, and 20.1. These last four, together with their paragraphs, should show like ordinary text.

This one is a little tricky, I think I have fixed it but best double-check.

Basically I've done it so that the "how is one biased by" questions are in a separate paragraph. To me this makes the structure clear. But if you want this sentence to join the following paragraph, let me know.

**Comment by sujato** [comment]
>The html is missing an open brace: "pli-tv-pvr17:40.8": "}

This must have been fixed, it is correct now.

**Comment by sujato** [comment]
> Add the html "

{}

" to the following segments:
pli-tv-bi-vb-pc73:1.16
pli-tv-bi-vb-pc75:1.16
pli-tv-bi-vb-pc64:1.16
pli-tv-bi-vb-pc72:1.14
pli-tv-bi-vb-pc63:1.13


While I do love a good mystery, I have no idea what to do here!

**Comment by sujato** [comment]
>From Tracy:

I'm not sure about this, but we've just fixed a bug here, so let's check the next production version.

**Comment by sujato** [comment]
FYI, I use "[logical punctuation](https://slate.com/human-interest/2011/05/logical-punctuation-should-we-start-placing-commas-outside-quotation-marks.html)" rules, which say that we should only include punctuation inside quotes if it pertains to the quote itself. The difference is most obvious in the case of questions:

- The book is called “The End of the World?” 
- The book is called “The End of the World”? 

In the first, the book title is a question, in the second, the person speaking is asking a question. It's not so important in other contexts, as it usually doesn't affect the meaning, but still I prefer the logical approach. (The MS Pali text, however, uses the American style.)

Thus:

- The Vinaya Piṭaka, the “Basket of Monastic Law,” contains the rules
- The Vinaya Piṭaka, the “Basket of Monastic Law”, contains the rules

Anyway this is just for your consideration, it's not a requirement.

**Comment by sujato** [comment]
Ok, I think I've done all these, with the caveats above.

Thanks so much, this is really precise and detailed work!

**Comment by brahmali** [comment]
> It's a bit of an odd structure, more like an Anguttara sutta in fact. But for me, I'd prefer it if we finished the rule with Yo vibhajeyya, āpatti thullaccayassa. What do you think?

OK. But let's at least fix the inconsistency by making segment 15.2.4 into a separate paragraph like segment 16.2.16.

> Basically I've done it so that the "how is one biased by" questions are in a separate paragraph. To me this makes the structure clear. But if you want this sentence to join the following paragraph, let me know.

It's not super important, but if the question is in a separate paragraph it will look different from the preceding and succeeding sections, which is not ideal. And so I would prefer "this sentence to join the following paragraph", with the same relationship to the bold heading as in the other sections.

> {}
> 
> " to the following segments:
> pli-tv-bi-vb-pc73:1.16
> pli-tv-bi-vb-pc75:1.16
> pli-tv-bi-vb-pc64:1.16
> pli-tv-bi-vb-pc72:1.14
> pli-tv-bi-vb-pc63:1.13
> 
> While I do love a good mystery, I have no idea what to do here!

Whoops! It should have said: `Add the html "<p class='subrule'>{}</p>" to the following segments:` I forgot to mark it as code. 

> FYI, I use "[logical punctuation](https://slate.com/human-interest/2011/05/logical-punctuation-should-we-start-placing-commas-outside-quotation-marks.html)" rules, which say that we should only include punctuation inside quotes if it pertains to the quote itself.

I've actually tried to follow this rule, but occasionally I may have slipped up.

Sādhu!

**Comment by sujato** [comment]
>But let's at least fix the inconsistency

Definitely, it should be consistent one way or another. And unless I have messed up, it is.

>it will look different from the preceding and succeeding sections

Indeed. But it's a bit more nuanced, because this section *is* different from the preceding and following ones. Namely, the definition part is longer. So for that reason it requires a few paragraphs, whereas the other sections don't have paragraphs. So while it is different, it is consistently  different: longer passages have paragraphs. It's just one of those things that come from the structure of the text, both highly logical but also highly elastic. 

Anyway, let's check how it appears and see if it's acceptable. It's not hard to change if I know what I'm after.

Gotta go, I'll fix the rest later.

**Comment by brahmali** [comment]
There are also some issues with the print version or rather the pdf. What should I do with these? Would you like a list here?

**Comment by brahmali** [comment]
Further changes to main introduction (as found on SC, [here](https://suttacentral.net/vinaya-guide-brahmali?lang=en)):

organisation → organization
The Khandhakas → the Khandhakas
bhikkhu pātimokkha → Bhikkhu-pātimokkha
pātimokkha” → Pātimokkha (17 times)

Publisher’s note: “Ajahn Brahmal’s”. An "i" would be nice!

**Comment by brahmali** [comment]
Here are a few suggestions from Ven. Nadi at Dhammasara:

"I assume the print version will eventually have a detailed table of contents rather than the very very top-level general one in the first version. That would take care of not having x.y.z numberings and the flipping through/figuring out structuriness."

"If the online table of contents had subheadings and subsubheadings and subsu … as well, that’d be dandy, but I’d weigh this less important than for the print version. It’s easier to scroll up and down a page (I have Kd 1 in mind! Lots of scrolling!) to check than flip physical pages."

"I like how in essays on Discourse, there’s that table of contents on the side that indicates where you are as you scroll through - that’d be super handy; I know that’s out of scope here, but since we’re sort of on the topic …"

"As long as you’re passing things along, then do please also pass along that I really appreciate how in the print version you can make all things disappear and be left with just text."

**Comment by brahmali** [comment]
Would you please add a heading at Kd 3, after segment 5.12.43:

"Others send a message"

**Comment by mtnadi** [comment]
Cleanup time!  Outstanding items from above, if I haven't missed anything:

---
~~To be merged in #4740 and #5322~~ Done! :
> Pvr.1.9:
> Indent verse lines: pli-tv-pvr1.9:4.1 + pli-tv-pvr1.9:13.1 (that is, add: )
> ...
> Pvr.2.9:
> Indent verse lines: pli-tv-pvr2.9:4.1 + pli-tv-pvr2.9:12.1 (that is, add: )

[150c6f0](https://github.com/suttacentral/bilara-data/commit/150c6f09b4c40528b98f3dc357c4d06b237c8698)

>  Pvr. 8:
> Move paragraph from after to before segment pli-tv-pvr8:4.2.

[b819af4](https://github.com/suttacentral/bilara-data/commit/b819af4ea374e370db1863d942e01e92e7b74f49)

> Would you please add a heading at Kd 3, after segment 5.12.43:
> 
> "Others send a message"

[2d76a51](https://github.com/suttacentral/bilara-data/commit/2d76a51777c12005fa8d2f74ee7aa3f2026b9965)


---
Still TODO:

>>    From Tracy:
>
> I'm not sure about this, but we've just fixed a bug here, so let's check the next production version.

So, links with segment anchors to headings (or when main references aren't displayed)!  Like `https://suttacentral.net/pli-tv-bu-vb-ss2/en/brahmali#3.7.0`.  They still don't work and it's not limited to the vinaya.  It's on my todo list to open a new ticket for this.

> Further changes to main introduction (as found on SC, [here](https://suttacentral.net/vinaya-guide-brahmali?lang=en)):
>
> organisation → organization
> The Khandhakas → the Khandhakas
> bhikkhu pātimokkha → Bhikkhu-pātimokkha
> pātimokkha” → Pātimokkha (17 times)
> 
> Publisher’s note: “Ajahn Brahmal’s”. An "i" would be nice!

~~These still need to be done.  Hopefully I can find out where to make these edits!~~ These have been done as of Aug 2026

**Comment by mtnadi** [comment]
following
> kd3: add heading after 5.12.43

segment numbers have since been aligned across all files. Insert new segment number (5.13.0) in comment, ref, root, variant files: fixed #5406 #5407

**Comment by brahmali** [comment]
> following
> 
> kd3: add heading after 5.12.43
> 
> segment numbers have since been aligned across all files. Insert new segment number (5.13.0) in comment, ref, root, variant files 

So, I take it you want me to split this segment and add the heading? I've just tried, and for some reason I am not able to split anything in Kd 3.

I get the following message: "Segments not found in root: html/pli/ms/vinaya/pli-tv-kd/pli-tv-kd3_html.json"

**Comment by mtnadi** [comment]
No, my apologies again; it's done, just housekeeping.  It wasn't something you could do in Bilara, and I see Hong Da also filled you in.  It's perfect that you got that message though!  Exactly the sort of message one can expect from satisfyingly robust software, ahhhh. :heart: bilara
